packages/hooks/src/useP2PAdvertList.ts

```diff
@@ -28,9 +33,47 @@ const useP2PAdvertList = (
         }));
     }, [data?.p2p_advert_list?.list, getRate]);
 
+    const loadMoreAdverts = React.useCallback((startIndex: number) => {
```

Comment:
Hi @jim-deriv , maybe you can consider using `useInfiniteQuery`, so you can handle loadmore, limit and offset easily 😄 , you can refer to `useTransactions` in `deriv/api/hooks`

Reply:
Hi @farhan-nurzi-deriv, Great Idea man, but if we keep on making changes like these, the card will never move forward. Ideally, this card was supposed to be delivered last sprint. It's only stack in code review. Let's consider this in a refactor card, as I'll need to test this again, which will take a lot of time for me to do

Reply:
Yeah alright, sure 🙏